packages/ketcher-core/src/application/editor/actions/closelyFusing.ts

```diff
@@ -98,3 +106,52 @@ function closestToMerge(struct, closestMap) {
 
   return mergeMap
 }
+
+function mergeItemsFuseMap(items: Map<number, number>): Set<number> {
+  const itemsSet = new Set<number>()
+  items.forEach((value, key) => {
+    itemsSet.add(value).add(key)
+  })
+  return itemsSet
+}
+
+function getAllConnectedAtomsIds(restruct, atomsIds, bondsIds) {
+  const srcAtoms = new Set(atomsIds)
```

Comment:
srcAtoms? atoms from the source? 🤔

Reply:
Source atoms. The atoms we are beginning with. I couldn't come up with a better naming. Any ideas?

Reply:
why not just 'atoms'?

Reply:
Because the result of the function is also "atoms". So basically we can name result "atoms" as well and we'll get function making atoms from atoms.
In order to differentiate them in some way we divide them into "source atoms" — the ones we are using for the process — and the resulting "connected atoms", which are the goal of the process.

Reply:
Isn't it 

```
() => {
 const atoms;
 ?
 return  ̶p̶r̶o̶f̶i̶t̶ connectedAtoms
}
```

?

Reply:
This way we have no clue about the purpose of this variable.
Are those target atoms? Are they going to be modified? Returned? What are we going to do with them?
Defining them as "source atoms" clearly states that it's an input to some process and they are not going to be used in any other way.

Reply:
maybe `initialAtoms` would be more straightforward then, what do you think?

Reply:
Do not see the difference, but ok, will rename it if this helps.